assets/app/view/game/buy_sell_shares.rb

```diff
@@ -272,6 +277,15 @@ def render_merge
 
         [h(:button, { on: { click: merge } }, 'Merge')]
       end
+
+      def render_convert
```

Comment:
is there a way to reuse the existing functionality like with abilities or choose?

Reply:
This is built on top of the existing functionality for converting corporations, the difference here is that it is a stock round action. As well as the normal selling or buying shares, you can convert a five-share corporation into a ten-share corporation. This UI is to give you a button to do this. Would using abilities make this any simpler?